### PR TITLE
feat: Add `initialScope` to `globalExtra`

### PR DESCRIPTION
**Pull request by timfish**
- Updates the e2e test server to support latest Sentry Relay features
- Adds `initialScope` as JSON to Electron `globalExtra` form parameters as `sentry___initialScope` 
- Adds a GPU crash test for Electron >= 13 with Electron uploader

### CODE REVIEW

File: src/main/backend.ts [hunk 1]
```diff
@@ -41,6 +41,26 @@ export async function isAppReady(): Promise<boolean> {
   );
 }
 
+/** Gets a Scope object with user, tags and extra */
+function getScope(): Partial<ScopeContext> | undefined {
```
Comment (timfish) [inline src/main/backend.ts:45]: Is there a better way than this to get an object with `user`, `tags` and `extra` from the scope?

Comment (kamilogorek) [reply src/main/backend.ts:45]: Not really, although I'd still rewrite it as:

```js
function getScope(): Partial<ScopeContext> {
  const scope = getCurrentHub().getScope() as any;

  if (!scope) {
    return {};
  }

  /* eslint-disable @typescript-eslint/no-unsafe-member-access */
  return {
    ...(scope._user && { user: scope._user }),
    ...(scope._tags && { tags: scope._tags }),
    ...(scope._extra && { extra: scope._extra }),
  };
}
```

It should be totally safe to always set initial scope to `{}` on line 220 then as well.

**Comment by timfish** [comment]
Looks like I need to do some debugging on Linux

**Comment by timfish** [comment]
@jan-auer I seem to have hit a rather strange bug in Breakpad where the `globalExtra` values get truncated at the first comma character which is quite a pain for JSON!

Do you want to just limit this feature to Crashpad platforms for now and we'll come back to this later?

**Comment by jan-auer** [comment]
Agreed, let's do just crashpad and then enable Linux when it has switched over.

**Comment by nornagon** [comment]
👋  thanks for this change! Super helpful to have the scope available on renderer crashes.

One more thing though—it seems like fields like `release` aren't automatically propagated here, so e.g. when searching for `release:xxxx`, all renderer crashes will be excluded. Would it be possible to include that data in `globalExtra` too?

**Comment by timfish** [comment]
Yes, good point. I'll get that added!

**Pull request merged**